framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -124,11 +130,11 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 		// TODO: CCM should be collected and stored via events
 		if (this._sidechainAPIClient) {
 			this._sidechainAPIClient.subscribe('chain_newBlock', async (data?: Record<string, unknown>) =>
-				this._newBlockhandler(data),
+				this._newBlockHandler(data),
 			);
 			this._sidechainAPIClient.subscribe(
 				'network_newBlock',
-				async (data?: Record<string, unknown>) => this._newBlockhandler(data),
+				async (data?: Record<string, unknown>) => this._newBlockHandler(data),
```

Comment:
Maybe it's outside of the PR, but it's not necessary to listen to the `network_newBlock` event. All the executed block will be emitted through `chain_newBlock` event